tasks/internal/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -118,49 +119,159 @@ spec:
           value: $(params.umb_url)
         - name: signature_data_file
           value: "$(params.signature_data_file)"
+        - name: CA_BUNDLE
+          value: /etc/pki/tls/certs/ca-bundle.crt
       script: |
         #!/usr/bin/env /bin/bash
         set -x
-        
-        MAX_RETRIES=3
-        RETRY_DELAY=5 # Initial delay
-        
-        set +x
-        # This helps with Shellcheck warning
-        echo "${UmbCert:?}" > /tmp/crt
-        echo "${UmbKey:?}" > /tmp/key
+        cat <<EOF > /tmp/pubtools-sign-config.yaml
+        msg_signer:
+          messaging_brokers:
+            - "${umb_url}"
+          messaging_cert_key: /tmp/umb.pem
+          messaging_ca_cert: ${CA_BUNDLE}
+          topic_send_to: ${umb_publish_topic}
+          topic_listen_to: queue://Consumer.${requester}.{task_id}.${umb_listen_topic}.{task_id}
```

Comment:
Where does task_id come from?

Reply:
task-id is cli argument accepted by pubtools-sign.